### ISSUE TITLE
suppression impossible avec la version 0.9

### ISSUE THREAD

**Issue by ghost** [alexisju/albinomouse-template#9]
Bonjour,
J'ai installé la version 0.9 avec ton thème et la suppression des entrées n'était plus possible. C'est sûrement lié au fait que le thème fasse une alerte.
En tout cas, merci beaucoup pour ce thème ;)

**Comment by alexisju** [comment]
Effectivement, le template n'a pas encore été mis à jour pour les dernières versions de Shaarli (> 8.3). Le problème est qu'à chaque mises à jour de Shaarli (dernièrement), les templates deviennent incompatibles et la maintenance laborieuse. 

La dernière version d'Albinomouse compatible est la version v0.8.3, en utilisant le même numéro de version de Shaarli.

J'envisage de "porter" le thème en pur CSS, mais pour ce faire, il faudrait que le thème par défaut spécifie davantage de classe et id CSS (cfr https://github.com/shaarli/Shaarli/issues/877#issuecomment-308147522). Ou que la maintenance des templates soit facilitée.

**Comment by ghost** [comment]
C'est dommage, j'ai installé la version 0.9 pour utiliser l'api (que je n'arrive toujours pas à utiliser d'ailleurs), et je n'ai pas de style qui me convienne mieux que le tien.

Si je comprend bien les URL pour les différentes actions changent, d'une version à une autre. Ce qui rend le javascript incompatible, d'où l'envie de faire une version en full CSS ?

Il resterait aussi d'autre problème comme des noms de variables non définies dans les templates, ici mes logs apache : 
`PHP Notice:  Undefined variable: timezone_form in /var/www/shaarli/tmp/configure.e0b7f24f55bb262d96486bec3362c3ca.rtpl.php on line 45, referer: https://shaarli.buron.coffee/?do=tools`
